atuin-client/src/database.rs

```diff
@@ -302,6 +330,49 @@ impl Database for Sqlite {
         Ok(res)
     }
 
+    async fn event_count(&self) -> Result<i64> {
+        let res: (i64,) = sqlx::query_as("select count(1) from events")
```

Comment:
I believe this works
```suggestion
        let res: i64 = sqlx::query_scalar("select count(1) from events")
```

Reply:
That's super neat! Thanks